app/redux/shared_actions/DeviceActionCreators.js

```diff
@@ -204,7 +205,7 @@ export const receivedLiftData = (isValid, data, time=new Date()) => (dispatch, g
     dispatch(TimerActionCreators.sanityCheckTimer());
     dispatch({
         type: ADD_REP_DATA,
-        isValid: isValid,
+        isValid: RepDataMap.isValidData(data),
```

Comment:
- [ ] why do we need this?

Reply:
I don't understand, is this not what you told me to do?

Reply:
isValidData is ALREADY BEING CALLED by someone. Look at who calls receivedLiftData.

If we want to do it here, that's fine, but we should then consolidate it here rather than calling it twice